include/infinispan/hotrod/RemoteCache.h

```diff
@@ -252,8 +253,8 @@ template<class K, class V> class RemoteCache: private RemoteCacheBase
      * \param key  key with which the specified value is to be associated
      * \param val  value value to be associated with the specified key
      * \param lifespan  lifespan of the entry.  Negative values are interpreted as unlimited lifespan.
-     *  \param lifespanUnit  unit of measurement for the lifespan
-     *  \param maxIdle  the maximum amount of time this key is allowed to be idle for before it is considered as
+     * \param lifespanUnit  unit of measurement for the lifespan
```

Comment:
Should this refer to `TimeUnit`, so users know which units are valid?

I think this is true of all of the `TimeUnit` parameters